docs/configuration/parameters.md

```diff
@@ -160,21 +178,32 @@ The list of acceptable values are:
 | Government cloud tenants (high) | gcchigh    |
 | Department of Defense tenants   | dod        |
 
-## KeepIndividualJSON
 
-**KeepIndividualJSON** Keeps the individual JSON files (e.g., `TeamsReport.json`) in the `IndividualReports` folder along with `ProviderSettingsExport.json` without combining the results in to one uber JSON file named the `ScubaResults.json`. The parameter is for backwards compatibility with older versions of ScubaGear.
+## NumberOfUUIDCharactersToTruncate
 
-| Parameter   | Value  |
-|-------------|--------|
-| Optional    | Yes    |
-| Datatype    | Switch |
-| Default     | n/a    |
-| Config File | No     |  
+**NumberOfUUIDCharactersToTruncate** controls how many characters will be truncated from the report UUID when appended to the end of **OutJsonFileName**.
+
+| Parameter   | Value              |
+|-------------|--------------------|
+| Optional    | Yes                |
+| Datatype    | Integer            |
+| Default     | 18                 |
+| Config File | Yes                |
```

Comment:
Add the default value for NumberOfUUIDCharactersToTruncate into full_config.yaml or some other sample config file.